common/djangoapps/student/tests/test_reset_password.py

```diff
@@ -252,6 +252,23 @@ def test_reset_password_good_token(self):
         self.user = User.objects.get(pk=self.user.pk)
         self.assertTrue(self.user.is_active)
 
+    def test_password_rest_fail(self):
+        """Tests that if we providing mismatching passwords user is not marked as active"""
```

Comment:
"Tests that if we providing mismatching passwords user is not marked as active" -> "Tests that if we provide mismatching passwords, user is not marked as active."